src/components/flyout/Flyout.scss

```diff
@@ -32,7 +32,7 @@
             transform: none !important;
         }
 
-        .overlay {
+        & > .flyout-overlay-content > .overlay {
```

Comment:
does this work?
```suggestion
        .flyout-overlay-content > .overlay {
```

Reply:
I updated it to add .overlay-content inside of the Overlay class instead. I didn't choose .overlay-wrapper because it conflicted with other css definitions

Reply:
To answer your question...yes that does work